mdaviz/mda_file_viz.py

```diff
@@ -44,26 +44,50 @@ def setMetadata(self, text, *args, **kwargs):
         # tab=self.metadataPage
         self.metadata.setText(text)
 
-    def setData(self, text, *args, **kwargs):
-        self.data.setText(text)
-
     def setPlot(self, plot_widget):
         layout = self.plotPageMpl.layout()
-        utils.removeAllLayoutWidgets(
-            layout
-        )  # TODO replace with mainWindow.clearContent?
+        # TODO - question: # why are we removing all the layouts? What are we doing here?
+        # TODO - later: replace removeAllLayoutWidgets with mainWindow.clearContent?
```

Comment:
What is removed exactly for each of these two methods?

Reply:
Tested that a bit, it is fine the way it is.